tests/functional/test_install_wheel.py

```diff
@@ -635,3 +635,126 @@ def test_correct_package_name_while_creating_wheel_bug(script, package_name):
     package = create_basic_wheel_for_package(script, package_name, '1.0')
     wheel_name = os.path.basename(package)
     assert wheel_name == 'simple_package-1.0-py2.py3-none-any.whl'
+
+
+def test_report_file_owner_conflicts(script, tmpdir):
+    """
+    Test installing from a wheel that wants to overwrite
+    files owned by another wheel to ensure that a warning
+    is reported and that the second wheel does overwrite
+    the first.
+    """
+    from tests.lib import TestFailure
```

Comment:
I see a few other tests in this file importing this inside the test. Maybe it's a good time to move this import along with the others at the top?

Reply:
I'm just following the local coding patterns in this PR. Changing all of that seems like a task for another PR.